diff --git a/ansible/environments/local/group_vars/all b/ansible/environments/local/group_vars/all
index b8e390cd..b0a409f6 100644
--- a/ansible/environments/local/group_vars/all
+++ b/ansible/environments/local/group_vars/all
@@ -48,3 +48,8 @@ env_hosts_dir: "{{ playbook_dir }}/environments/local"
 container_pool_akka_client: true
 runtimes_enable_concurrency: true
 limit_action_concurrency_max: 500
+
+limit_invocations_per_minute: 60000
+limit_invocations_concurrent: 30000
+limit_fires_per_minute: 60000
+limit_sequence_max_length: 50
diff --git a/ansible/roles/controller/tasks/deploy.yml b/ansible/roles/controller/tasks/deploy.yml
index 64724c41..1e8e3347 100644
--- a/ansible/roles/controller/tasks/deploy.yml
+++ b/ansible/roles/controller/tasks/deploy.yml
@@ -342,7 +342,7 @@
     ports: "{{ ports_to_expose }}"
     # userns_mode, pid_mode and privileged required when controller running in lean mode
     userns_mode: "{{ userns_mode | default('') }}"
-    pid_mode: "{{ pid_mode | default('') }}"
+    # pid_mode: "{{ pid_mode | default('') }}"
     privileged: "{{ privileged | default('no') }}"
     command:
       /bin/sh -c
diff --git a/core/controller/src/main/scala/org/apache/openwhisk/core/controller/actions/SequenceActions.scala b/core/controller/src/main/scala/org/apache/openwhisk/core/controller/actions/SequenceActions.scala
index 619ccdbd..8286c70d 100644
--- a/core/controller/src/main/scala/org/apache/openwhisk/core/controller/actions/SequenceActions.scala
+++ b/core/controller/src/main/scala/org/apache/openwhisk/core/controller/actions/SequenceActions.scala
@@ -244,7 +244,7 @@ protected[actions] trait SequenceActions {
         Parameters(WhiskActivation.kindAnnotation, JsString(Exec.SEQUENCE)) ++
         causedBy ++ waitTime ++ binding ++
         sequenceLimits,
-      duration = Some(accounting.duration))
+      duration = Some(java.time.Duration.between(start, end).toMillis))
   }
 
   /**
